src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+  const stylesArray = sourceString.split(';');
+  // розрізали на ("властивість: значення")
+
+  stylesArray.forEach((style) => {
+    const [property, ...valueParts] = style.split(':');
```

Comment:
[CODE KNOWLEDGE] - When splitting the 'style' string, you can directly destructure it into 'property' and 'value' without using the rest syntax '...valueParts' since you are expecting only two parts. This will simplify the code and make it more readable.